web/components/chrome.jsx

```diff
@@ -58,6 +58,9 @@ class Chrome extends React.Component {
 				{ this.renderGlobalAlert() }
 				<div className="container">
 					{ this.props.children }
+					<div className="text-right footer">
```

Comment:
Is `footer` a CSS class? Did I actually make that or did I accomplish everything with `text-right`, `<small>`, and `<em>`?